AppCenterData/AppCenterData/Internal/Model/MSPageInternal.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSPage.h"
+
```

Comment:
You need the nullability declaration in this file too. Please use `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`.